lib/spack/spack/ci.py

```diff
@@ -2147,39 +2146,49 @@ def process_command(name, commands, repro_dir):
     return exit_code
 
 
-def create_buildcache(**kwargs):
+def create_buildcache(
+    input_spec: spack.spec.Spec,
+    pr_pipeline: bool,
+    pipeline_mirror_url: Optional[str] = None,
+    buildcache_mirror_url: Optional[str] = None,
+) -> List[PushResult]:
```

Comment:
In case we can:
```suggestion
def create_buildcache(
    input_spec: spack.spec.Spec,
    *,
    pr_pipeline: bool,
    pipeline_mirror_url: Optional[str] = None,
    buildcache_mirror_url: Optional[str] = None,
) -> List[PushResult]:
```
to force callers to use keyword arguments (for readability).